feat(react-grid): add scroll to row in virtual table (#2511)
DevExpress/devextreme-reactive: packages/dx-react-grid/docs/reference/virtual-table.md

stubCellComponent | ComponentType&lt;[Table.CellProps](table.md#tablecellprops)&gt; | | A component that renders a stub table cell if the cell value is not provided.
stubHeaderCellComponent | ComponentType&lt;[Table.CellProps](table.md#tablecellprops)&gt; | | A component that renders a stub header cell if the cell value is not provided.
messages? | [Table.LocalizationMessages](table.md#localization-messages) | | An object that specifies the localization messages.

> onTopRowChange? | (rowId: number &#124; string) => void | | Handles top row index and id changes.

Reviewer: ```suggestion
onTopRowChange? | (rowId: number &#124; string) => void | | Handles a change of the top row.
```

## Methods

Name | Type | Description
-----|------|------------

> scrollToRow | (rowId: number &#124; string) => void | A function, that scroll table to specified row.

Reviewer: ```suggestion
scrollToRow | (rowId: number &#124; string) => void | Scrolls table to a row with the specified ID.
```

stubCellComponent | ComponentType&lt;[Table.CellProps](table.md#tablecellprops)&gt; | | A component that renders a stub table cell if the cell value is not provided.
stubHeaderCellComponent | ComponentType&lt;[Table.CellProps](table.md#tablecellprops)&gt; | | A component that renders a stub header cell if the cell value is not provided.
messages? | [Table.LocalizationMessages](table.md#localization-messages) | | An object that specifies the localization messages.

> onRowIndexChange? | (index: number) => void | | Handles top row index changes.

Reviewer 2: We should consider combining these events into one with args = { index, rowId } because their purpose is the same

onRowIdChange? | (id: number &#124; string) => void | | Handles top row ID changes.

## Methods

Name | Type | Description
-----|------|------------

> scrollToIndex | (index: number) => void | A function, that scroll table to specified index.

Reviewer 2: The same goes for scrollTo.. methods - we can replace them with `scrollToRow`

COLUMN_TYPE | symbol | The data column type's indentifier.
ROW_TYPE | symbol | The data row type's indentifier.
NODATA_ROW_TYPE | symbol | The nodata row type's indentifier.

> TOP_POSITION | symbol | The top table position for scroll.

Reviewer: ```suggestion
TOP_POSITION | symbol | The top position of the table. Used in scrolling.
```

> BOTTOM_POSITION | symbol | The bottom table position for scroll.

Reviewer: ```suggestion
BOTTOM_POSITION | symbol | The bottom position of the table. Used in scrolling.
```
